### ISSUE TITLE
overflow menu color issue.

### ISSUE THREAD

**Issue by kk63852** [fossasia/phimpme-android#2516]
**Actual Behaviour**

when theme is changed to dark amoled, overflow menu color stays white.

**Expected Behaviour**

it should change to dark when theme is changed to dark amoled.

**Screenshots of the issue**

![whatsapp image 2019-02-11 at 11 53 10](https://user-images.githubusercontent.com/31561661/52548501-16d77580-2df4-11e9-8a47-8953d0a7bfc8.jpeg)

**Would you like to work on the issue?**

yes.

**Comment by yashk2000** [comment]
@kk63852 the menu color does change. The app just has to be restarted. You should make a snackbar show up after theme  change saying that all changes will take place after the app is restarted.

**Comment by kk63852** [comment]
is it really working. it doesn't seem to work on my device even after restarting?

**Comment by kk63852** [comment]
this issue seems to be happening in trash bin and upload history and only when the theme is set to dark amoled.

**Comment by yashk2000** [comment]
Yes I checked again. Whenever the theme is changes, overflow menu colors change only after restarting the app.

**Comment by kk63852** [comment]
ok. i'll check on some other device later on and then i'll make changes to the issue after confirming it. thanks anyway.

**Comment by yashk2000** [comment]
It is in trashbin and upload history, text is also not visible. It should be fixed.

**Comment by kk63852** [comment]
text is visible. its just the background that does not change color.

**Comment by neel1998** [comment]
@kk63852  @yashk2000  I am having the same issue even after restarting the app after changing the theme to DARK AMOLED. The background of overflow menu in upload history is not changing

**Comment by yashk2000** [comment]
Yes. This issue is not fixed. @kk63852 is working on it.

**Comment by neel1998** [comment]
okay

**Comment by yashk2000** [comment]
@kk63852 are you still working on this issue? If not, can I please work on it?

**Comment by yashk2000** [comment]
@kk63852 any response?

**Comment by yashk2000** [comment]
Due to inactivity on this issue, I'm taking it up.

**Comment by kk63852** [comment]
i had internals going on. ended today. will try to be active from tomorrow. meanwhile you are free to work on this issue as you already referenced a pr. @yashk2000

**Comment by yashk2000** [comment]
It wasn't a pr..it was a similar issue that was opened. If you are working on this, continue, else I can work on it.

**Comment by kk63852** [comment]
> It wasn't a pr..it was a similar issue that was opened. If you are working on this, continue, else I can work on it.

feel free to work on it.
